assets/src/containers/ResourcesAccessed.js

```diff
@@ -81,6 +81,14 @@ function ResourcesAccessed (props) {
         </div>
       )
     }
+    else if (resourceTypes.length === 1) {
+      let message = "You are viewing " + resourceTypes[0] + " data"
```

Comment:
This is `const`

Reply:
on Line 48 in `ResourcesAccessed.js`, can you change `resourceTypes` variable from `const` to `let`?